vignettes/rromeo.Rmd

```diff
@@ -15,54 +15,166 @@ vignette: >
+  rromeo::rr_publisher_name(name = "Oxford", qtype = "all")
+  rromeo::rr_publisher_name(name = "Oxford University", qtype = "all")
+  rromeo::rr_publisher_name(name = "Oxford University", qtype = "exact")
+  rromeo::rr_publisher_id(id = 55)
+  rromeo::rr_publisher_country(country = "IR")
+  rromeo::rr_publisher_continent(continent = "Australasia")
+}, record = "once")
 
 insert_cassette("vignette_cassette")
 ```
 
-```{r}
-library(rromeo)
-```
-
-# Get summary data
+## Query Journal Data
 
 **NOTE:** SHERPA/RoMEO data is released under the [Creative Commons Attribution-NonCommercial-ShareAlike 2.5 (CC BY-NC-SA 2.5) license](https://creativecommons.org/licenses/by-nc-sa/2.5/). A suggestion of citation is included in `rromeo` via `citation("rromeo")`
 
-Let's try to access summary data for . We can access this data using its full
-name or its unique ISSN identifier.
+The SHERPA/RoMEO database contains information on the archival policies of academic journals and publishers. Some journals let you archive a version of a submitted manuscript or book chapter (= preprint) before it's reviewed, some also let you archive the peer-reviewed but not formatted version (= postprint) and some even let you archive the peer-reviewed **and** formatted version of the manuscript (= pdf or publisher's version). The goal of `rromeo` is to make this database accessible through R.
+
+### Query using ISSN
+
+Let's try to access summary data using the ISSN of a journal:
+
+```{r query_issn}
+library(rromeo)
 
-```{r}
 rr_journal_issn("1947-6264")
 ```
 
-You can also search for partial matches.
+From this we see that the archival of the preprint is permitted while some restrictions apply for both postprint and pdf version of the manuscript. The restrictions are visible in the `*_embargo` field: the authors have to wait 12 months after publication before archiving postprint and formatted manuscript.
+
 
-```{r}
-rr_journal_name("Biogeography", qtype = "contains")
+### Query by Journal Title
+
+We can also search journal by names using `rr_journal_name()` the first argument `name` is the string to look for and the second argument `qtype` defines the way to match. If we want to look at journals that have the word `Biostatistics` in their title we can use the following code:
+
+```{r query_name_contains}
+rr_journal_name("Biostatistics", qtype = "contains")
 ```
 
-If rromeo finds multiple journals matching your query terms, it will only return
+Because `rromeo` finds multiple journals matching your query terms, it will only return
 the list of titles and ISSN by default.
-
 You can then use this list to select the exact journal you are looking for (in
 this case, it is recommended to use the ISSN).
-
 Alternatively, you may want to get data for all those journals. To achieve this,
-you may enable the `multiple` option:
+you may enable the `multiple` argument to `TRUE`:
 
-```{r}
+```{r query_name_contains_multiple}
```

Comment:
(Not our fault but `&` matches `and`. It's not super intuitive.)

Reply:
Should we mention it somewhere?